conversation.go

```diff
@@ -12,10 +12,14 @@ type conversation struct {
 }
 
 // GroupConversation is the foundation for Group and Channel
-type groupConversation struct {
-	conversation
+type GroupConversation struct {
```

Comment:
This shouldn't be exported. The fields within groupConversation are exported and can be accessed when embedded.